spritzle/tests/test_resume_data.py

```diff
@@ -47,8 +47,13 @@
     with open(torrent_dir / 'random_one_file.torrent', mode='rb') as f:
         torrent_info = lt.torrent_info(lt.bdecode(f.read()))
     core.session.add_torrent({'ti': torrent_info})
+    # We need time for the torrent added alert to come in to initiate save
```

Comment:
I wonder if it would better to add an alert manager handler to know exactly when the resume data has been saved instead of sleeping. We could probably wait for save_resume_data_alert before continuing.